src/Features/LanguageServer/Protocol/Handler/SemanticTokens/SemanticTokensRefreshQueue.cs

```diff
@@ -82,14 +73,12 @@ public Task OnInitializedAsync(ClientCapabilities clientCapabilities, RequestCon
             && clientCapabilities.Workspace?.SemanticTokens?.RefreshSupport is true
             && _capabilitiesProvider.GetCapabilities(clientCapabilities).SemanticTokensOptions is not null)
         {
-            // Only send a refresh notification to the client every 2s (if needed) in order to avoid
-            // sending too many notifications at once.  This ensures we batch up workspace notifications,
-            // but also means we send soon enough after a compilation-computation to not make the user wait
-            // an enormous amount of time.
+            // Only send a refresh notification to the client every 2s (if needed) in order to avoid sending too many
+            // notifications at once.  This ensures we batch up workspace notifications, but also means we send soon
+            // enough after a compilation-computation to not make the user wait an enormous amount of time.
             _semanticTokenRefreshQueue = new AsyncBatchingWorkQueue<Uri?>(
                 delay: TimeSpan.FromMilliseconds(2000),
-                processBatchAsync: (documentUris, cancellationToken)
-                    => FilterLspTrackedDocumentsAsync(_lspWorkspaceManager, _notificationManager, documentUris, cancellationToken),
+                processBatchAsync: FilterLspTrackedDocumentsAsync,
                 equalityComparer: EqualityComparer<Uri?>.Default,
                 asyncListener: _asyncListener,
                 _disposalTokenSource.Token);
```

Comment:
i had missed the event registration 2 lines below this.  so we still def want to keep thsi queue.